Run.py

```diff
@@ -432,6 +433,12 @@ def FlavEffList(Flavor, Output, Dataset, bdiscDirectory, Save):
     ApplyJER = True
 #    ---------------------------------------------------------------------------------------------------------------------    #
  
+elif args.jec:
```

Comment:
This is great.  I will consider re-writing the btagging corrections to give all three systematics similar to this for the sake of consistency (and it'll be easier in the long run to have fewer coffea output files).